src/otx/algorithms/common/utils/__init__.py

```diff
@@ -63,3 +63,12 @@
 if is_hpu_available():
     os.environ["PT_HPU_LAZY_MODE"] = "1"
     import habana_frameworks.torch.gpu_migration  # noqa: F401
+
+
+if is_xpu_available():
+    import mmcv
```

Comment:
Because anomaly task doesn't use `mmcv`, I think we need to execute only if `mmcv` can be imported.

Reply:
How can we check it? try-except?

Reply:
yes I think that's a best way we have.

Reply:
Done.